server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/mail/Email.scala

```diff
@@ -61,7 +62,18 @@ object Email {
   }
 
   def from(message: (MessageId, Seq[MessageResult])): Email =
-    Email(message._1, sanitizeSize(message._2.head.getSize))
+    Email(message._1, asEmailHeaders(message._2.head.getHeaders.headers()), sanitizeSize(message._2.head.getSize))
+
+  def asEmailHeaders(headers: java.util.Iterator[Header]): Seq[EmailHeader] = {
```

Comment:
private def ...

Reply:
Don't use Java collections in a scala method signature. Maybe passing `message._2.head.getHeaders` as an argument will be better?